File: Data Structures & Algorithms/sort-colors/submission-0.js
class Solution {
    /**
     * @param {number[]} nums
     * @return {void} Do not return anything, modify nums in-place instead.
     */
    sortColors(nums) {
        const colorCount = [ 0, 0, 0]
        for ( const num of nums) {
            colorCount[num]++
        }

        let currentIdx = 0;
        for ( let count = 0; count < colorCount[0]; count++) {
            nums[currentIdx++] = 0
        }

        for ( let count = 0; count < colorCount[1]; count++) {
            nums[currentIdx++] = 1
        }

        for ( let count = 0; count < colorCount[2]; count++) {
            nums[currentIdx++] = 2
        }
    }


}

File: Data Structures & Algorithms/jump-game/submission-0.py
class Solution:
    def canJump(self, nums: List[int]) -> bool:
        canreach = [False for i in range(len(nums))]
        canreach[-1] = True

        for i in range(len(nums) - 2, -1, -1):
            for j in range(i + 1, i + nums[i] + 1):
                if canreach[j]:
                    canreach[i] = True
                    break
            
        print(canreach)
        return canreach[0]